masterlist.yaml

```diff
@@ -949,6 +960,22 @@ plugins:
         itm: 840
         udr: 82
         nav: 59
+      # v1.6.1170.0
+      - <<: *reqManualFix
+        crc: 0xD399544A
+        util: '[SSEEdit v4.1.5d](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
+        itm: 842
+        udr: 82
+        nav: 59
+      - <<: *reqManualFix
+        crc: 0xA62C1C10
+        util: '[SSEEdit v4.1.5d](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
+        itm: 7
+        nav: 59
+      - <<: *reqManualFix
+        crc: 0xA5AB2927
+        util: '[SSEEdit v4.1.5d](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
+        nav: 59
```

Comment:
These can be removed.

Reply:
When there are two passes performed in xEdit (as is uniquely the case for Dawnguard.esm it seems), shouldn't the itm/udr be summed for Pass 1 and Pass 2? So ...
```
      - <<: *reqManualFix
        crc: 0xD399544A
        util: '[SSEEdit v4.1.5d](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
        itm: 842
        udr: 82
        nav: 59
      - <<: *reqManualFix
        crc: 0xA62C1C10
        util: '[SSEEdit v4.1.5d](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
        itm: 7
        nav: 59
```
would be ...
```
      - <<: *reqManualFix
        crc: 0xD399544A
        util: '[SSEEdit v4.1.5d](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
        itm: 849
        udr: 82
        nav: 59
```

Reply:
I don't think that doing this is necessary. By adding only the first (unchanged) result from cleaning the file we already have accomplished that the user is informed about the plugin being dirty.